.github/workflows/linkcheck.yml

```diff
@@ -5,6 +5,7 @@ name: Markdown Link Check
 jobs:
   markdown-link-check:
     runs-on: ubuntu-22.04
+    continue-on-error: true # can fail for external reasons, so don't be a blocker
```

Comment:
the external reason would be that a website returns a 404 error? 

could it fail for a "internal" reason as well?

Reply:
I can't see a way to differentiate between these two. When the link is dead there's no way to tell if the link is wrong, or the server temporary down.

IIUC this change will mark the job as passed, so we'll see in the UI that there are some bad links, but it won't block the CI on it.

Reply:
ok, sounds good